stanford_paragraph_types.features.filter.inc

```diff
@@ -18,6 +18,15 @@ function stanford_paragraph_types_filter_default_formats() {
     'status' => 1,
     'weight' => 0,
     'filters' => array(
+      'pathologic' => array(
+        'weight' => -50,
+        'status' => 1,
+        'settings' => array(
+          'local_paths' => 'https://sites.stanford.edu/jsa-content/

```

Comment:
These shouldn't be in the feature as they get overridden so easily.
Remove the pathologic config completed and replace with an alter.

eg: https://github.com/SU-SWS/stanford_jumpstart/blob/7.x-5.x/stanford_jumpstart.module#L285